src/Main.java

```diff
@@ -0,0 +1,42 @@
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        //1.Initiate a Set. Using For loop add integers from 1 to 10.
+        Set<Integer> test = new HashSet<>();
+        for (int i = 1; i < 11; i++) {
+            test.add(i);
+        }
+        System.out.println(test);
+
+        //2.Remove all odd numbers from sets.
+        int m = 0;
+        List<Integer> convert = new ArrayList<>(test);
+
+        for (int j = 0; j < convert.size(); j++) {
+            if (convert.get(j) % 2 != 0) {
+                convert.remove(j);
+                j -= m;
+                m++;
+            }
+        }
```

Comment:
You have to remove items from the set ("test") and please use for it "removeIf" method or use iterator.